server/services/prisonService.ts

```diff
@@ -15,4 +15,35 @@ export default class PrisonService {
   async deactivatePrisonService(token: string, prisonId: string, serviceCode: ServiceCode) {
     return this.prisonApiClient.servicePrisons.deactivatePrisonService(token, { prisonId, serviceCode })
   }
+
+  async getScreenStatus(token: string, prisonId: string) {
+    return this.prisonApiClient.splashScreen.getScreenStatus(token, { prisonId, moduleName: 'OIDCHOLO' })
+  }
+
+  async addCondition(token: string, prisonId: string, block: boolean = true) {
+    return this.prisonApiClient.splashScreen.addCondition(
+      token,
+      {
+        prisonId,
+        moduleName: 'OIDCHOLO',
+      },
+      {
+        conditionType: 'CASELOAD',
+        conditionValue: `${prisonId}`,
+        blockAccess: block,
+      },
+    )
+  }
+
+  async removeCondition(token: string, prisonId: string) {
+    return this.prisonApiClient.splashScreen.removeCondition(token, { prisonId, moduleName: 'OIDCHOLO' })
+  }
+
+  async updateScreen(token: string, prisonId: string, block: boolean) {
+    return this.prisonApiClient.splashScreen.updateScreen(token, {
+      prisonId,
+      moduleName: 'OIDCHOLO',
+      blockScreen: block === true ? 'true' : 'false',
```

Comment:
See prior comment

```suggestion
      blockScreen: block,
```

Reply:
Disregard, technically it would be handled correctly but typescript won't allow it